client/src/templates/Challenges/components/hotkeys.tsx

```diff
@@ -67,8 +66,8 @@ interface HotkeysProps {
   submitChallenge: () => void;
   innerRef: MutableRefObject<HTMLElement | undefined>;
   instructionsPanelRef?: React.RefObject<HTMLElement>;
-  nextChallengePath: string;
-  prevChallengePath: string;
+  nextChallengePath: string | null;
+  prevChallengePath: string | null;
```

Comment:
because we are using challengeMeta from the store, can you link this type to the redux type? 

This will make sure that people who refactor in the future will align the types across the application if they types are linked

```suggestion
```

```ts
interface HotkeysProps & Pick<ChallengeMeta, "nextChallengePath" | "prevChallengePath"> {
```

Reply:
Good idea, thanks!